### DISCUSSION TITLE
Why my Github Pages settings show nothing other than Upgrades?

### DISCUSSION THREAD

**Discussion by yfucityu** [community/community#123617] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![Pages](https://github.com/community/community/assets/107677340/f089a63b-4a42-443c-937e-2d2ee9f25109)

**Comment by Alireza-Valizadeh** [comment, marked as answer]
Github pages for public deployment is available on the "Free" plan. The problem is because the repository you are trying to deploy is "Private" therefore you see this page. Private Deployment is only available for the "Enterprise" plan -> 
[_NOTE:  To publish a GitHub Pages site privately, your organization must use GitHub Enterprise Cloud._](https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site#about-access-control-for-github-pages-sites)

**Reply by Alireza-Valizadeh** [reply]
You either need to make your repository public or upgrade your account 😉
